iphone/Classes/TiUIWebViewProxy.m

```diff
@@ -412,27 +423,103 @@ - (void)repaint:(id)unused
 
 - (void)goBack:(id)unused
 {
-  [[[self webView] webView] goBack];
+  [[self wkWebView] goBack];
+}
+
+#if IS_SDK_IOS_14
+- (void)findString:(id)args
+{
+  NSString *searchString = [args objectAtIndex:0];
+  ENSURE_STRING(searchString);
+  KrollCallback *callback;
+  NSDictionary *options = nil;
+  if ([args count] > 2) {
+    options = [args objectAtIndex:1];
+    ENSURE_DICT(options);
+    callback = [args objectAtIndex:2];
+  } else {
+    callback = [args objectAtIndex:1];
+  }
+  ENSURE_TYPE(callback, KrollCallback);
+
+  if (![TiUtils isIOSVersionOrGreater:@"14.0"]) {
+    [callback call:@[ @{ @"success" : NUMBOOL(NO), @"error" : @"Supported on iOS 14+" } ] thisObject:self];
```

Comment:
As mentioned in the docs, probably want to use the `[TiUtils dictionaryWithCode:]` for all these callback args (here, and the code below) so we are consistent about the shape of our callback response objects.